packages/core/helper-plugin/package.json

```diff
@@ -41,7 +41,7 @@
     "test:front:cov": "run -T jest --config ./jest.config.front.js --coverage",
     "test:front:watch": "run -T jest --config ./jest.config.front.js --watchAll",
     "watch": "cross-env NODE_ENV=development webpack-cli -w",
-    "lint": "run -T eslint ."
+    "lint": "run -T eslint . --ext .js,.jsx,.tsx,.ts"
```

Comment:
can be configured in the eslint settings directly I believe

Reply:
Yeah I couldn't find this one out for the global setting, do you know which one it is? :(

Reply:
I don't think that is possible either :)

> This request came up multiple times before. Unfortunately, as @mysticatea explained, it's not really possible to do, especially, when you add complexity of shareable configs and cascading into the picture. So I'm going to close this issue, sine we can't address this.

https://github.com/eslint/eslint/issues/11223

Reply:
yeah it won't pick up `.jsx` files for some reason 🤔  it'll do `.ts`, `.tsx` & `.js` but not `.jsx`....

Reply:
ho ok that's strange. I usually just set this up https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md